azurerm/internal/services/logic/logic_app_workflow_resource.go

```diff
@@ -271,6 +290,11 @@ func resourceArmLogicAppWorkflowRead(d *schema.ResourceData, meta interface{}) e
 				d.Set("workflow_version", v["contentVersion"].(string))
 			}
 		}
+		var integrationAcc string
+		if props.IntegrationAccount != nil && props.IntegrationAccount.ID != nil {
+			integrationAcc = *props.IntegrationAccount.ID
```

Comment:
Would this look better if we called `d.Set("logic_app_integration_account_id", integrationAcc)` in his if block rather than outside of it?

Reply:
Hi @mbfrahry ,Sorry that it'll cause the `ImportStateVerify` Error, I think we need always set an empty string on this field:
```
Step 7 error: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.
        
        (map[string]string) {
        }
        
        
        (map[string]string) (len=1) {
         (string) (len=32) "logic_app_integration_account_id": (string) ""
        }
        
FAIL
```

Reply:
Hey @yupwei68, we can instead add a bit to the import test step to get around this. 
```
data.ImportStep("logic_app_integration_account_id"),
```

That will ignore any weird import things that happen to that attribute